src/response.ts

```diff
@@ -0,0 +1,33 @@
+import idpayguard from './js/idpayguard';
+import { initHeader } from './js/header';
+import { setTranslateBtns } from './js/translateui';
+
+// eslint-disable-next-line sonarjs/cognitive-complexity
+document.addEventListener('DOMContentLoaded', async () => {
+  // TO UNCOMMENT
+  // document.body.classList.add('loadingOperations');
+
+  // idpayguard
+  idpayguard();
+
+  // initHeader
+  initHeader();
+
+  // init translations
+  setTranslateBtns();
+
+  // set email address in placeholder
+  const useremailPlaceholder = document.querySelectorAll('.windowcont__response__useremail') || null;
+  const useremailArray = Array.from(useremailPlaceholder);
+  const useremail = sessionStorage.getItem('useremail') || '';
+  for (const el of useremailArray) {
+    // eslint-disable-next-line functional/immutable-data
+    (el as HTMLElement).innerText = useremail;
+  }
+  // clear sessionStorage
+  sessionStorage.clear();
+
+  // CALL TO PM HERE
```

Comment:
In another PR, the call that verifies the transaction status must be added here?